src/kirby/components/list/list.component.tns.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Component,
+  ContentChild,
+  ContentChildren,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  QueryList,
+  TemplateRef,
+} from '@angular/core';
+import {
+  LoadOnDemandListViewEventData,
+  RadListView,
+} from 'nativescript-ui-listview/ui-listview.common';
+
+import {
+  ListItemDirective,
+  ListHeaderDirective,
+  ListSectionHeaderDirective,
+  ListCellDirective,
+} from './list.directive';
+
+@Component({
+  selector: 'kirby-list',
+  templateUrl: './list.component.html',
+  styleUrls: ['./list.component.scss'],
+})
+export class ListComponent implements OnInit {
```

Comment:
Do yourself a favor and extend `ContentView`, and register element with typescript (it will integrate more nicely, behave in layouts, etc.)

See: https://github.com/kirbydesign/designsystem/blob/master/src/kirby/components/card/card.component.tns.ts (class declaration and bottom of file)